LaterPay/Wrapper.php

```diff
@@ -74,4 +74,15 @@ public static function laterpay_remote_get( $url, $args = array(), $fallback_val
 
         return $response;
     }
+
+    /**
+     * Get LaterPay Plugin Version.
+     *
+     * @return string
+     */
+    public static function laterpay_version() {
+        if ( function_exists( 'get_option' ) ) {
+            return get_option( 'laterpay_plugin_version' );
+        }
```

Comment:
in case of non-wp use of lib, we should return the `0.3` version. 

Also, update the version from 0.2 to 0.3 on any other place required on this lib?  @thrijith